catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/web/metrics/MemoryMetricsStorage.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright 2023 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics;
+
+import java.time.Instant;
+import org.apache.iceberg.metrics.MetricsReport;
+
+/** Store Iceberg metrics in memory, used for test */
+public class MemoryMetricsStorage implements IcebergMetricsStorage {
```

Comment:
This can be moved to test if it is only used for test. Instead, you need to have a BlackholeMetricsStore as the default one.

Reply:
done